experiments/long_runs/leaderboard/leaderboard.jl

```diff
@@ -0,0 +1,165 @@
+import ClimaAnalysis
+import GeoMakie
+import CairoMakie
+import Dates
+
+include("data_sources.jl")
+
+"""
+    compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+
+Plot the biases against observations and simulation data.
+
+The paramter `leaderboard_base_path` is the path to save the leaderboards and bias plots and
+`diagnostics_folder_path` is the path to the simulation data.
+"""
+function compute_leaderboard(leaderboard_base_path, diagnostics_folder_path)
+    @info "Error against observations"
+
+    # Set up dict for storing simulation and observational data after processing
+    sim_obs_comparsion_dict = Dict()
+
+    # Print dates for debugging
+    _, var_func = first(sim_var_dict)
+    var = var_func()
+    output_dates =
+        Dates.DateTime(var.attributes["start_date"]) .+
+        Dates.Second.(ClimaAnalysis.times(var))
+    @info "Working with dates:"
+    @info output_dates
+
+    short_names = keys(sim_var_dict)
+    for short_name in short_names
+        @info short_name
+        # Simulation data
+        sim_var = sim_var_dict[short_name]()
+
+        # Observational data
+        obs_var = obs_var_dict[short_name](sim_var.attributes["start_date"])
+
+        # Remove first spin_up_months months from simulation
+        spin_up_months = 12
+        spinup_cutoff = spin_up_months * 30 * 86400.0
+        ClimaAnalysis.times(sim_var)[end] >= spinup_cutoff && (
+            sim_var =
+                ClimaAnalysis.window(sim_var, "time", left = spinup_cutoff)
+        )
+
+        # Get 12 or less months of data
```

Comment:
Why is this here? What if we had run for many years, and only used the first year as spinup?

Reply:
I added that there in case if the simulation run for longer than one year, but less than two years. For instance, if one were to run the simulation for 1.5 years, then only six months will be used after throwing away the first year. 

The code will always use the second year regardless if the simulation run for multiple years. It assumed that we start a year 2012 and end at year 2013 (although, it could be any year as long as data exists in the observations) The only reason is because some datasets are limited in time. For example, for `et`, the last point in time is `2013-12-16`.